### PR TITLE
Branch 1

### PR DESCRIPTION
**Pull request by gonjavi**
Things than in this pull request:

- Created todo list
- User can create projects
- Users can create todos inside each project, with properties: title, description, due date, and the priority.
- Used materialize-CSS for styling

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "javascripttodolist",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack",
+    "watch": "webpack --watch"
```
Comment (Oghenebrume50) [inline package.json]: You should have a more professional setup of Webpack on package.json 
`"scripts": {`
    `"dev": "webpack --mode development",`
    `"build": "webpack --mode production",`
    `"start": "webpack-dev-server --mode development --open" // optional`
  `},`
install necessary packages for this
`

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,183 @@
+// import { stringify } from 'querystring';
```
Comment (Oghenebrume50) [inline src/index.js]: It’s better to delete commented code. If you need to review a previous version of your code, use git history see this
please do this for all other occurrences

File: src/index.js [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+  if (selectedPro === '' || title === '' || description === '' || duedate === '') {
+    ok = false;
+  }
+  const alert = document.getElementById('alert');
+  if (ok === false) {
+    alert.innerHTML = 'Please enter all the information for the Todo\n';
+    return ok;
+  }
+  addNewTodo(selectedPro, title, description, duedate, priority);
+  // alert.innerHTML = '';
```
Comment (Oghenebrume50) [inline src/index.js]: delete commented code

File: src/modules/projects.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+          priority: 'Important and urgent',
+        },
+        {
+          title: 'Next year',
+          description: 'Third todo project 1',
+          duedate: 'Mar 19, 2021',
+          priority: 'Important',
+        },
+      ],
+    },
```
Comment (Oghenebrume50) [inline src/modules/projects.js]: This is not needed you don't need to create content in the default project, that being said there should be only one default project see tis

**Comment by gonjavi** [comment]
Once again, how it is said here at the last point: https://github.com/microverseinc/linters-config/tree/master/javascript

Stickler does not work and nothing helps boom - run eslint in your local env and correct all errors. Remember to let your Code Reviewer know that you had problems with Stickler and you used linter in local env.

I do not have errors in local env.

Thanks

File: package.json (hunk 1 above)
Comment (gonjavi) [reply package.json]: Perfect

File: src/index.js (hunk 2 above)
Comment (gonjavi) [reply src/index.js]: perfect

File: src/index.js (hunk 3 above)
Comment (gonjavi) [reply src/index.js]: perfect

File: src/modules/projects.js (hunk 4 above)
Comment (gonjavi) [reply src/modules/projects.js]: perfect

**Comment by gonjavi** [comment]
the app allows:
- create and delete projects
- create, edit and delete todos, which are saved inside projects.

**Comment by MauricioRobayo** [comment]
Hi again @gonjavi !

I am really sorry but still facing the same issue after your last commit:

todo-app (0)

Here's the [live link](https://cdn.statically.io/gh/gonjavi/JavascriptTodoList/ca7abbdb8c3943022f4f7b8abfa96c4bf7703dec/dist/index.html) I tested.

**Comment by gonjavi** [comment]
It is working well if you want to display the todos saved, you need to click on the project to display them.

Regards

**Comment by gonjavi** [comment]
Thanks

**Pull request merged**